### PR TITLE
Add tabs & overflow menu

### PR DESCRIPTION
**Pull request by IMcD23**
Sorry for the relatively large change here, it contains the following:

- Add [TabView](https://github.com/IMcD23/TabView) CocoaPod
- Replace root view controller of the app with a `TerminalTabViewController`, containing `TerminalViewController` instances (previously `ViewController`)
- Remove `ViewController` from Storyboard, layout with code
- Added overflow logic, based on size class, for bar button items in the `TerminalViewController` to overflow into a popover (issue #67)
- Added updated icons (and their SVG files):
    - Add (new tab)
    - Bookmarks (anticipating #65)
    - History
    - Open (for document browser)
    - Script
    - Settings

Limitations:
- Can't run multiple commands at the same time in multiple tabs. The next command dispatched will not run until the previous one completes. This is because we need to switch the process's working directory to the one for the tab each time a command is run.

### CODE REVIEW

**Comment by IMcD23** [comment]
@louisdh This is feature complete now, but it will have conflicts with #63 and #65, so perhaps we wait until those are checked in? Not sure on the timeline for those, so it's up to you.

File: OpenTerm/AppDelegate.swift [hunk 1]
```diff
@@ -16,7 +16,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 		// Override point for customization after application launch.
 
+        window = UIWindow(frame: UIScreen.main.bounds)
+        window?.rootViewController = TerminalTabViewController()
 		window?.tintColor = .defaultMainTintColor
+        window?.makeKeyAndVisible()
```
Comment (BasThomas) [inline OpenTerm/AppDelegate.swift]: Indentation?

**Comment by IMcD23** [comment]
@louisdh this is good to merge now. Merge conflicts are resolved, and I made sure basic scenarios are working.

Please run `swiftlint autocorrect --format --use-tabs` after merging to fix indentation issues.

**Comment by louisdh** [comment]
One more thing before merging: I'm sorry but I'm not really a fan of the .afdesign format that the icons are in. Can you export them to a more open format? Either SVG if they're vector or simply PNG if they're bitmaps. Alternatively I would also accept PSD. The simple reason for this: I don't have Affinity Designer, so I wouldn't be able to open the icon files if I merge this in the current state 😢.

**Comment by IMcD23** [comment]
Done.

**Comment by louisdh** [comment]
Thank you!

**Pull request merged**